python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -214,6 +215,16 @@ def get_schema(cls):
                     minimum: 0
                 default: 2.
 
+              time_on_target:
+                description: Override the automatically calculated time_on_target value used
+                    to optimize the telescope cable wrap positions.
+                    Set to zero (default) to calculate automatically.
+                anyOf:
+                  - type: "null"
+                  - type: number
+                    minimum: 0.0
```

Comment:
wait, you want the minimum to be null no, so it will compute the value instead of using 0?